frontend/components/FightHistory.js

```diff
@@ -0,0 +1,126 @@
+            },
+            {
+                eventID: "1213123324",
+                time: "02:33",
+                referee: "Main referee",
+                decision: "BLUE: defense, miss, 0 points IDK."
+            },
+        ])
+        // console.log(events);
+    }, [])
+
+    return (
+        <>
+            <FormHeader name="Fight History"></FormHeader>
+            <Center>
+                <VStack width="90%">
+                    <Center>
+                        <HStack width="100%" mb="20px">
+                            <Box bg="red.500" p="5px" width="50%" borderColor="black" borderWidth="1" height="40px">
+                                <Center>
+                                    <Text color="white">{props.state.fighter1}</Text>
+                                </Center>
+                            </Box>
+                            <Box bg="blue.500" p="5px" width="50%" borderColor="black" borderWidth="1" height="40px">
+                                <Center>
+                                    <Text color="white">{props.state.fighter2}</Text>
+                                </Center>
+                            </Box>
+                        </HStack>
+
+                        <DisplayScore
+                            fighter1Score={props.state.fighter1Score}
+                            fighter2Score={props.state.fighter2Score}>
+                        </DisplayScore>
+
+                    </Center>
+                </VStack>
+
+                <Box id="oncoming_matches" height="auto" width="90%">
+                    <Text fontSize={20} marginBottom={3}>
+                        <Center>
+                            Event decisions
+                        </Center>
+                    </Text>
+                    {/* TODO: update filtering when model will contain bool determining if fight is over / or maybe score is not null? */}
+                    <FlatList backgroundColor="gray.200"
+                        data={events} renderItem={
+                            ({ item }) =>
+                                <Box borderBottomWidth={1} borderTopWidth={1} _dark={{ borderColor: "gray.800" }} borderColor="coolGray.400" pl="4" pr="5" py="2">
+                                    <HStack space={3} justifyContent="space-between">
+                                        <Text _dark={{ color: "warmGray.50" }} color="coolGray.800" bold>
+                                            {item.referee + ": "}
+                                        </Text>
+                                        <Spacer />
+                                        <Text fontSize="xs" _dark={{ color: "warmGray.50" }} color="coolGray.800" alignSelf="flex-start">
+                                            {item.time}
+                                        </Text>
+                                    </HStack>
+                                    <Text _dark={{ color: "warmGray.50" }} color="coolGray.800" alignSelf="flex-center">
```

Comment:
App running on Expo breaks, we have to change 'flex-center' to 'center'
```suggestion
                                    <Text _dark={{ color: "warmGray.50" }} color="coolGray.800" alignSelf="center">
```